src/chapters/booster-rockets/bestpractices.rst

```diff
@@ -0,0 +1,232 @@
+strong idea of how to convert those words into working code.
+
+.. admonition:: Take-home lesson:
+
+   Be ready for day one on the job.  *Do your homework*.
+
+Personal Drive (Grit)
+---------------------
+
+Does the following statement resonate with you?  If so, perfect.
+
+   "If I stumble, I WILL pick myself up, brush off the dust, and try again."
+
+All throughout LC101, you will be asked to code.  Each exercise, studio and
+assignment is designed to give you experience through practice.  You WILL make
+mistakes, and that is OK.  Often, our mistakes teach us more than getting the
+"correct" answer on our first try.
+
+Every genius programmer you see on Facebook or YouTube started out in front of
+a screen saying, “Oops,” “ARGH!” or “#*&%%@#!”  No one simply “gets” coding
+without some trial-and-error.
+
+.. admonition:: Take-home lesson:
+
+   Use your mistakes as learning opportunities.  *Do your homework*.
+
+Effort = Outcome
+-----------------
+
+| Let’s take a look at a sample coding task:
+| *“Prompt the user to enter a number, then print ‘Even’ if it is divisible
+   by 2, otherwise print ‘Odd’.”*
+
+Now let’s take a look at an imaginary student's attempt at solving this
+problem:
+
+.. sourcecode:: javascript
+
+   1.
+   2.
+   3.
+   4.
+   5.
+
+Hmmm.  A blank answer space.  What might be the reason?
+   a. The student did not understand how to solve the problem.
+   b. The student knew how to solve the problem and decided to skip the task.
+   c. The student tried to solve the problem, could not get the program to work,
+      so deleted the code.
+   d. The student ran out of time when trying to complete the prep work before
+      class.
+
+From a teacher’s perspective, ANY of these reasons could be valid, and we have
+no way of determining which is true.  This prevents us from knowing how to best
+help the student.  Where would be begin?
+
+For the student, a blank response provides no benefit because the necessary
+practice was either ignored or incomplete.  Students gain only as much as they
+put in. SO:
+
+a. Even if you have no clue how to approach a task, MAKE AN ATTEMPT ANYWAY,
+   then ask questions.
+b. If you know how to solve the problem, COMPLETE THE TASK ANYWAY, because
+   practice makes better.  Also, you could use your code to help answer a
+   classmate’s question.
+c. If you tried to solve the problem, but your code did not work, DO NOT DELETE
+   YOUR ATTEMPT.  Ask a question.  Showing your work to your teacher, TA or
+   classmates will give them a clear idea about your thought process.
+d. If you ran out of time, GO BACK AND FILL IN THE BLANKS LATER.  Practice
+   makes better.  If you neglect one set of skills, then the tasks that come
+   later and depend on those skills will be more difficult.
+
+.. admonition:: Take-home lesson:
+
+   Learning takes work, and you need the practice.  *Do your homework*.
+
+Sports Motivational Stuff
```

Comment:
Per discussion: Look for non-semantic visual ways to differentiate between these quotes (list-table, list, etc).

Reply:
Placed quotes into a table with alternate shaded rows.